municipal_finance/resources.py

```diff
@@ -149,3 +154,55 @@ class AgedCreditorItemsV2Resource(resources.ModelResource):
     class Meta:
         model = AgedCreditorItemsV2
         import_id_fields = ['code']
+
+
+class IncexpFactsV1Resource(resources.ModelResource):
```

Comment:
I know I'm a stuck record but I really think facts resources should be kept in a clearly tests-only part of the codebase so that no one accidentally thinks these are ok to use in production

Reply:
@jbothma I moved these here because they are now shared between the 'scorecard' and 'municipal_finance' apps. It feels right to me to have these defined in the application itself, as it's a utility related to the app, but I'll move it to the 'tests' directory of the 'municipal_finance' app if you prefer.

Reply:
I do prefer. It's test-only code - you can have test utility code shared between apps. Thanks

Reply:
@jbothma no worries. moved.